dockerfiles/Dockerfile.audit

```diff
@@ -0,0 +1,11 @@
+FROM ubuntu:20.04 as ci
```

Comment:
License Finder has its own dockerfile which already includes all dependencies and the tool itself — https://hub.docker.com/r/licensefinder/license_finder

I think we should use that so we don't need to write custom dockerfiles for each repo

Reply:
Done, although we still need to build our own image `FROM` it as there are some extra dependencies we need.